src/metabase/models/pulse.clj

```diff
@@ -192,7 +193,7 @@
   "Update an existing `Pulse`, including all associated data such as: `PulseCards`, `PulseChannels`, and `PulseChannelRecipients`.
 
    Returns the updated `Pulse` or throws an Exception."
-  [{:keys [id name cards channels] :as pulse}]
+  [{:keys [id name cards channels skip_if_empty?] :as pulse}]
```

Comment:
The other argument to that function `creator_id` should actually be `creator-id` as well, if you want to change it for consistency that would be 👍